tests/test_configlib.py

```diff
@@ -98,16 +99,31 @@ def test_ConfigIniEnv(datadir):
     assert cie.get('foo', namespace='namespacebaz') == 'bat'
 
 
+def test_config_ini_file_does_not_exist():
+    with mock.patch('os.environ') as os_environ_mock:
+        os_environ_mock.__contains__.return_value = True
+        os_environ_mock.get.return_value = 'doesnotexist.ini'
+        print os_environ_mock.mock_calls
+        print os_environ_mock.method_calls
```

Comment:
Rrrr... I removed it, but didn't save the file. Sorry about that.

As a side note, `mock_calls` is super helpful when you're trying to figure out wtf got called so you can mock it properly.